eZ/Publish/Core/Limitation/LanguageLimitationType.php

```diff
@@ -187,12 +194,8 @@ private function evaluateObject(ValueObject $object, APILimitationValue $value):
         if ($object instanceof Content) {
             $object = $object->getVersionInfo();
         } elseif ($object instanceof ContentInfo) {
-            try {
-                $object = $this->persistenceContentHandler->loadVersionInfo(
-                    $object->id,
-                    $object->currentVersionNo
-                );
-            } catch (NotFoundException $e) {
+            $object = $this->loadVersionInfo($object);
+            if ($object === null) {
                 return self::ACCESS_DENIED;
             }
         }
```

Comment:
> @alongosz `in_array` with 3rd argument is part of the phpstan strict rules extension.
> 
> https://github.com/phpstan/phpstan-strict-rules
> 
> We could add these rules on the case-by-case basis, because some directly contradict our current style (for example disallowed `empty()`)

Fully agree, but it seems to be out of scope for a Customer Request.